Makefile

```diff
@@ -9,4 +9,5 @@ test:
 clean:
 	dune clean
 
-.PHONY: all clean
+format:
+	ocamlformat --inplace $(git ls-files '*.ml' '*.mli')
```

Comment:
Is this the same as `dune build --auto-promote @fmt`? If so, I prefer that.

Reply:
Certainly, I didn't know about it